docs/source/dialogue.rst

```diff
@@ -8,6 +8,71 @@ The figure below shows the dialogue flow in our system.
 
 .. image:: _static/dialogue_flow.png
 
+Intents
+-------
+
+User intents
+"""""""""""""
+
+:py:class:`UserIntents <moviebot.intents.user_intents>`
+
++------------------------+------------+
+| Intent   | Description   |
++========================+============+
+| Reveal   | The user wants to reveal a preference.   | 
++------------------------+------------+
+| Inquire  | Once the agent has recommended an item, the user can ask further details about it.        | 
++------------------------+------------+
+| Remove preference  | The user wants to remove any previously stated preference.        | 
++------------------------+------------+
+| Reject  | The user either has already seen/consumed the recommended item or does not like it.        | 
++------------------------+------------+
+| Accept  | The user accepts (likes) the recommendation. This will determine the success of the system as being able to find a recommendation the user liked.        | 
++------------------------+------------+
+| Continue recommendation  | If the user likes a recommendation, they can either restart, quit or continue the process to get a similar recommendation.        | 
```

Comment:
To my understanding there are different intents for restart and quit. This one is specifically a request for a new recommendation (e.g., *Recommend me something similar*).

Reply:
After looking at the code, I see that the NLG method `generate_output` calls [_user_options_continue](https://github.com/iai-group/MovieBot/blob/bf39d971d7dffb1c3f0532dd416e577abb72a2c6/moviebot/nlg/nlg.py#L457) to propose several options, i.e, restart, quit, or similar recommendation.